src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTargetSpelling.java

```diff
@@ -227,7 +227,7 @@ private void injectContextMenuHandler()
          }
          word = docDisplay_.getTextForRange(wordRange);
 
-         if (word == null || word.length() < 2 || typoSpellChecker_.checkSpelling(word))
+         if (word == null || typoSpellChecker_.checkSpelling(word))
```

Comment:
This was just removing an inconsistency where the spellchecker would underline something but the context menu wasn't triggering.